backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/predicate/impl/ElasticsearchSimpleQueryStringPredicateBuilder.java

```diff
@@ -63,6 +66,38 @@ public void defaultOperator(BooleanOperator operator) {
 		}
 	}
 
+	@Override
+	public void enable(SimpleQueryFlag operation) {
+		if ( this.operations == null ) {
+			this.operations = EnumSet.noneOf( SimpleQueryFlag.class );
+		}
+		this.operations.add( operation );
+	}
+
+	@Override
+	public void enable(EnumSet<SimpleQueryFlag> operations) {
+		if ( this.operations == null ) {
+			this.operations = EnumSet.noneOf( SimpleQueryFlag.class );
+		}
+		this.operations.addAll( operations );
+	}
+
+	@Override
+	public void disable(SimpleQueryFlag operation) {
+		if ( this.operations == null ) {
+			this.operations = EnumSet.noneOf( SimpleQueryFlag.class );
+		}
+		this.operations.remove( operation );
+	}
+
+	@Override
+	public void disable(EnumSet<SimpleQueryFlag> operations) {
+		if ( this.operations == null ) {
+			this.operations = EnumSet.noneOf( SimpleQueryFlag.class );
+		}
+		this.operations.removeAll( operations );
+	}
+
```

Comment:
The "operations" are not used. We need to implement support for flags for Elasticsearch.

Reply:
It is possible that they support this flag or support will appear in Elasticsearch in the future. They also follow "Apache Lucene". He can simply ignore this possibility in Elasticsearch until they solve this problem.